src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/NavigationBar.tsx

```diff
@@ -0,0 +1,217 @@
+import * as React from 'react';
+import { Grid, makeStyles, createMuiTheme } from '@material-ui/core';
+import { useDispatch, useSelector } from 'react-redux';
+import { IRuntimeState, Triggers } from 'src/types';
+import { FormLayoutActions } from 'src/features/form/layout/formLayoutSlice';
+import { getTextResource } from '../../utils/formComponentUtils';
+import { AltinnAppTheme } from 'altinn-shared/theme';
+import { useState, useEffect } from 'react';
+
+const theme = createMuiTheme(AltinnAppTheme);
+
+const useStyles = makeStyles({
+  ul: {
+    marginBottom: '6px!important',
+    listStyleType: 'none',
+    textDecoration: 'none!important' as 'none',
```

Comment:
I don't really understand what's actually going on here. Why adding `as 'none'`
Are you using some sort of type checking plugin in your IDE?